Tasks/Npm/Tests/test-commandWithoutArguments.ts

```diff
@@ -0,0 +1,20 @@
+import ma = require('vsts-task-lib/mock-answer');
+import tmrm = require('vsts-task-lib/mock-run');
+import path = require('path');
+import util = require('./NpmMockHelper');
+
+let taskPath = path.join(__dirname, '..', 'npmtask.js');
+let taskMockRunner = new tmrm.TaskMockRunner(taskPath);
+let npmMockHelper = new util.NpmMockHelper(taskMockRunner, "root", "");
+
+npmMockHelper.useDeprecatedTask();
```

Comment:
I'd rather not duplicate them. In order to pass something to the scripts I'd need a change in vsts-task-lib so the runner can take some args that get passed along.